### PR TITLE
Align style source along with design

### PR DESCRIPTION
**Pull request by TrySound**
Ref https://github.com/webstudio-is/webstudio-builder/issues/807 https://github.com/webstudio-is/webstudio-builder/issues/1108

- more consistent input spacing
- more visible text editing
- consistent border around input instead of shadow
- toggle selected state

## Code Review

- [ ] hi @rpominov, I need you to do
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-builder/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

**Review by rpominov — COMMENTED** [review]
Didn't notice anything else

**Comment by TrySound** [comment]
Fixed both

File: apps/builder/app/builder/features/style-panel/style-source/style-source-input.tsx [hunk 1]
```diff
@@ -64,18 +64,31 @@ export type ItemSelector = {
...
+  alignItems: "center",
+  backgroundColor: theme.colors.backgroundControls,
+  gap: theme.spacing[3],
+  p: theme.spacing[3],
+  borderRadius: theme.borderRadius[4],
+  minHeight: theme.spacing[12],
+  minWidth: 0,
+  border: `1px solid ${theme.colors.borderMain}`,
+  "&:focus-within": {
+    outline: `2px solid ${theme.colors.borderFocus}`,
```
Comment (rpominov) [inline apps/builder/app/builder/features/style-panel/style-source/style-source-input.tsx:80]: Need `outlineOffset: -1` to hide the grey the border:
 
Screenshot 2023-05-06 at 12 10 21

**Review by rpominov — APPROVED** [review]

File: apps/builder/app/builder/features/style-panel/style-source/style-source-input.tsx (hunk 1 above)
Comment (TrySound) [reply apps/builder/app/builder/features/style-panel/style-source/style-source-input.tsx:80]: Nice!

**Pull request merged**